packages/editing-adapter/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "npm run build:tsc && npm run build:rollup",
     "build:tsc": "tsc --incremental && cp src/gltf/*.d.ts lib/gltf",
     "build:rollup": "rollup -c --environment NODE_ENV:production",
-    "clean": "rm -rf lib && rm -rf dist",
+    "clean": "rm tsconfig.tsbuildinfo; rm -rf lib && rm -rf dist",
```

Comment:
Why is this specific to these two packages?

Reply:
Well, the only other package that uses --incremental is modelviewer...but for some reason that's not an issue? May be a question for @cdata